src/game-support/weird-west-de.md

```diff
@@ -0,0 +1,16 @@
+# Weird West: Definitive Edition
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Platinum installs=Yes opens=Yes}}
```

Comment:
Downgrade to `Bronze` due to the configuration need before gameplay and dense areas stuttering.